data/docs/instrumentation/opentelemetry-nextjs.mdx

```diff
@@ -78,7 +78,9 @@ export function register() {
 **Step 4.** Once you're done configuring the exporter options, try running your application and validate if your application is sending traces to SigNoz cloud [here](#validating-instrumentation-by-checking-for-traces).
 
 <Admonition>
-The instrumentation file should be in the root of your project and not inside the app or pages directory. If you're using the src folder, then place the file inside src alongside pages and app.
+- The instrumentation file should be in the root of your project and not inside the app or pages directory. If you're using the src folder, then place the file inside src alongside pages and app.
+
+- For older versions of NextJS(14 below), the [experimental instrumentation hook](https://nextjs.org/docs/14/pages/api-reference/next-config-js/instrumentationHook) needs to be enabled
```

Comment:
14 and below

Also add this 

module.exports = {
  experimental: {
    instrumentationHook: true,
  },
}

Reply:
Ok